package.json

```
@@ -69,6 +70,8 @@
     "graphql": "^14.7.0 || ^15.8.0"
```

To match the Apollo Server:

```suggestion
    "graphql": "^15.3.0 || ^16.0.0"
```

- https://www.npmjs.com/package/apollo-server?activeTab=code